boefjes/clients/bytes_client.py

```diff
@@ -111,7 +111,7 @@ def get_normalizer_meta(self, normalizer_meta_id: str) -> NormalizerMeta:
 
     @retry_with_login
     def save_raw(
-        self, boefje_meta_id: str, raw: bytes, mime_types: Set[str] = None
+        self, boefje_meta_id: str, raw: bytes, mime_types: Union[Set[str] , None] = None
```

Comment:
Nice find! Perhaps this fits better as an `Optional` type rather than a `Union`, since `Optional[Set[str]]` resolves to `Union[Set[str] , None]` and at the same time this is a 'nullable' argument but not an optional argument.

```suggestion
        self, boefje_meta_id: str, raw: bytes, mime_types: Optional[Set[str]] = None
```